src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -47,13 +47,24 @@ export class CalcitePickListItem {
       this.emitChangeEvent();
     }
   }
-
+  /**
+   * The icon for the pick-list-item. If no icon to be used, null is passed.
+   */
   @Prop({ reflect: true }) icon: ICON_TYPES | null = null;
 
+  /**
+   * The heading for the pick-list-item.
+   */
   @Prop({ reflect: true }) textHeading: string;
 
+  /**
+   * The description for the pick-list-item.
+   */
   @Prop({ reflect: true }) textDescription: string;
 
+  /**
+   * The value of the pick-list-item.
+   */
```

Comment:
Perhaps we can put a note on placement or note why/how it's used. Most of these comments don't provide more info than the prop name at the moment.